src/background/lotus-client/Lotus.js

```diff
@@ -465,6 +465,61 @@ class Lotus {
       signature: decoded[10].toString('hex')
     }
   }
+
+  async sendFunds(amount, toWallet) {
+    ports.postLog(`DEBUG: lotus.sendFunds(): sending ${amount} to ${toWallet}`);
+
+    //
+    // Get nonce
+    //
+    let nonce = await this.getNonce(this.wallet);
+    nonce = nonce.result;
```

Comment:
You're erroring here.  I don't think this line is needed, but here's how you would do it if you wanted it:  `ports.postLog(`${nonce}`);`  Reason:  postLog() can only take a string.

Reply:
Fixed